introduction-exercises.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: Introduction Exercises
+permalink: /introduction-exercises/
+---
+## Slides
+
+  [pdf](https://github.com/potassco-asp-course/exercises/releases/download/v1.0/introduction.pdf)
+  [git](https://github.com/potassco-asp-course/exercises)
+  [v1.0](ihttps://github.com/potassco-asp-course/exercises/tree/v1.0)
```

Comment:
```suggestion
  [v1.0.0](https://github.com/potassco-asp-course/exercises/tree/v1.0)
```

Reply:
Also: Does it ake sense to have both `intro.md` and `intro-ex.md` ...? Shouldn't these be one?